assets/stylesheets/components/header.scss

```diff
@@ -0,0 +1,3 @@
+header {
```

Comment:
Components must always use a class name. Targets HTML tags directly for `base/` styles.